.travis/before_install.sh

```diff
@@ -53,55 +47,45 @@ black --check --diff .
 # Lint code.
 flake8 --config flake8.cfg
 
-cd ..
-git clone --depth=1 https://github.com/pulp/ansible-pulp.git
-if [ -n "$PULP_ROLES_PR_NUMBER" ]; then
-  cd ansible-pulp
-  git fetch --depth=1 origin +refs/pull/$PULP_ROLES_PR_NUMBER/merge
-  git checkout FETCH_HEAD
-  cd ..
-fi
+# check for imports from pulpcore that aren't pulpcore.plugin
+#./.travis/check_pulpcore_imports.sh
```

Comment:
Found it. We need to make sure that the `SingleContentArtifactField` is properly exported in the plugin api...